trimmer_decorator.rb

```diff
@@ -3,6 +3,6 @@
 # creating a class that inherits Decorator class features
 class TrimmerDecorator < Decorator
   def correct_name
-    @nameable.correct_name[0, 10]
+    @nameable.correct_name[0..9]
   end
```

Comment:
- [ ] The change you  made  simply takes the first 10 characters using `@nameable.correct_name[0..9].` If the length is 10 or less, it returns the original string as well, without any additional check.

- [ ] However, we can use a ternary operator (? :) to check if the length is greater than 10. If it is, it takes the first 10 characters; otherwise, it returns the original string. This means that if the length is 10 or less, the original string is returned without modification.This will go well with the project requirements

### Requirement:
![image](https://github.com/IvonneBenitesRodriguez/oop-school-library/assets/50990925/acc26600-917d-4c7c-8de3-c7c67cfd7b54)

### Sample code:
```ruby
def correct_name
      @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
    end
```

- In practical terms, However, if there's a possibility that @nameable.correct_name might be nil or not a string, the change i suggested above is safer because it checks for the existence of `@nameable.correct_name` and handles non-string cases by returning the original value.

Reply:
Dear @Shedrack-Sunday 
I made the change you suggested according your specifications, 
please can you verify it, since it is almost my last PR, and I am concerned about it.
thanks in advance,
Best regards,